doc/reference/configuration/cfg_binary_logging_snapshots.rst

```diff
@@ -111,3 +112,28 @@
     | Default: 16777216 bytes
     | Environment variable: TT_WAL_QUEUE_MAX_SIZE
     | Dynamic: **yes**
+
+.. _cfg_binary_logging_snapshots-wal_cleanup_delay:
+
+.. confval:: wal_cleanup_delay
+
+    Since version :doc:`2.6.3 </release/2.6.3>`.
+    The delay (in seconds) used to prevent the :ref:`Tarantool garbage collector <cfg_checkpoint_daemon-garbage-collector>`
+    from immediately removing :ref:`write-ahead log<internals-wal>` files after a node restart.
+    This delay eliminates possible erroneous situations when the master deletes WALs
+    needed by :ref:`replicas <replication-roles>` after restart.
+    As a consequence, replicas sync with a master faster after its restart and
+    don't need to download all the data again.
+
+    Once all the nodes in the replica set are up and running,
+    automatic cleanup is started again even if ``wal_cleanup_delay`` has not expired.
+
+    .. NOTE::
+
+        The ``wal_cleanup_delay`` option is not in effect if a node is running as an
```

Comment:
If I get it right , this wording will be more precise:

```suggestion
        The ``wal_cleanup_delay`` option has no effect on nodes running as
```

I mean, it still has effect on other nodes in the replica set except anonymous replicas, right?

Reply:
Thanks, sounds better.